modules/administration-guide/pages/enabling-fuse-for-all-workspaces.adoc

```diff
@@ -7,6 +7,11 @@
 [id="enabling-fuse-overlayfs-for-all-workspaces"]
 = Enabling fuse-overlayfs for all workspaces
 
+For Podman 5.x, the `/home/user/.config` must be owned by the current user for Podman to function correctly.
+The `storage.conf` file for Podman is typically stored in this folder.
+This document explains how to configure the container entrypoint script for the workspace so that fuse-overlayfs is being used.
+The Universal Developer Image (UDI) already contains the necessary configuration by default.
```

Comment:
when you refer to the configuration already contained in UDI, what sort of configuration are you referring to? the one from the previous line (container entrypoint script)?

Reply:
Yes, the container entrypoint script, for example:
https://github.com/devfile/developer-images/blob/e24cbd7d58b8a744467db2e404314b63c30d8f85/base/ubi9/entrypoint.sh#L8-L16